web/book/src/project/contributing/development.md

```diff
@@ -88,12 +88,14 @@ since it relies on `brew`.
   [Taskfile.yml](https://github.com/PRQL/prql/blob/main/Taskfile.yml) will be a
   good source to copy & paste instructions from.
 
-### Option 3: Use a [Dev Container](https://containers.dev/)
+### Option 3: Use a Dev Container
 
 This project has a
 [devcontainer.json file](https://github.com/PRQL/prql/blob/main/.devcontainer/devcontainer.json)
 and a
 [pre-built dev container base Docker image](https://github.com/PRQL/prql/pkgs/container/prql-devcontainer-base).
+Dev Containers are specified at
+[https://containers.dev/](https://containers.dev/)
```

Comment:
Link is great but maybe we don't need this now you added it to the title?

Reply:
Actually, I took the link _out_ of the title/heading because there was no link icon when I viewed the Markdown. (Those link icons were present for Options 1, 2, and 4)

But when looking at the Book directly either one looks good. Let's leave this like this.